Makefile

```diff
@@ -771,6 +771,8 @@ install.docker-full: install.docker install.docker-docs
 .PHONY: install.systemd
 ifneq (,$(findstring systemd,$(BUILDTAGS)))
 install.systemd:
+	# Make sure that the service files point to the correct install path of Podman.
+	grep -lr "/usr/bin/podman" contrib/systemd | xargs sed -i 's@/usr/bin/podman@$(DESTDIR)$(BINDIR)/podman@g'
```

Comment:
Suggestion: `xargs --no-run-if-empty`. 

OTOH, if somehow/someday `/usr/bin/podman` is removed/changed in `contrib/systemd`, perhaps we *want* this to fail and get a heads-up. I would prefer a better error message than `sed: no input files` though.

Reply:
And, to do this right, you can't do `sed -i`. You'll need to rename all the source files to `.in`, and change `/usr/bin/podman` to `@@PODMAN@@`, and add a new `systemd-files` target with a rule that replaces `@@PODMAN@@`, and then install those generated files (and add them to `.gitignore`). This is becoming heavyweight very quickly.

Reply:
Yeah, sounds like lots more ways to break.  What about using `awk` with a `@@PODMAN@@` token in the source files, and output-redirecting to the destination directly (instead of using `install`)?